src/lib.rs

```diff
@@ -669,7 +669,14 @@ impl Library {
                         self.include_paths.push(PathBuf::from(inc));
                     }
                 }
-                _ => (),
+                _ => {
+                    let path = std::path::Path::new(part);
+                    if path.is_file() {
+                        // Pass file paths directly to the linker instead of going through
+                        // rustc adding platform-specific prefixes/suffixes with rustc-link-lib.
+                        config.print_metadata(&format!("rustc-link-arg={}", part));
```

Comment:
It seems problematic that this is not available via the API in any way. Maybe we need to add something like `self.libs` but for full paths to libraries here, but how would that even be called? `link_files`, similar to `link_paths`?

Do you have any suggestions?

Reply:
And then it would also be easy to add a test :)

Reply:
That would be inconsistent because the existing libs are just library names (with missing file extension, missing `lib` prefix etc) while these here are full paths.